mantidproject/mantid
# 6.5 release notes images and updates

**Description of work.**

- Added images to 
  - `driffaction.rst`
  - `direct_geometry.rst`
  - `indirect_geometry.rst`
  - `mantidworkbench.rst`
  - `reflectometry.rst`
- Created a new location from release note images to help organise the images directory - `docs/source/images/6_5_release`
- Moved some new release notes to their respective `used` folders and added them to the main files
  - `34436.rst`
  - `34433.rst`
  - `34395.rst`
  - `34400.rst`
- Added a link to `diffraction.rst`
- Reformatted a link in `indirect_geometry.rst`


**To test:**



- Remove build/docs/doctrees
- Build the documents again and check there are no warnings
- Check that the links work
- Check the information is still correct
- Grammar/Spelling check
- Check images are displaying properly


*There is no associated issue.*




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
